meta-lmp-base/classes/lmp.bbclass

```diff
@@ -56,3 +56,66 @@ IMAGE_CMD_ostree_append () {
 	# Update default home path
 	sed -i -e 's,:/home,:/var/rootdirs/home,g' usr/etc/passwd
 }
+
+run_fiotool_cmd () {
+	if [ -n "${SOTA_PACKED_CREDENTIALS}" ]; then
+                if [ -e "${SOTA_PACKED_CREDENTIALS}" ]; then
+                        "${1}" -repo "${OSTREE_REPO}" -creds "${SOTA_PACKED_CREDENTIALS}"
+                else
+                        bbwarn "SOTA_PACKED_CREDENTIALS file does not exist."
+                fi
+        else
+                bbwarn "SOTA_PACKED_CREDENTIALS not set. Please add SOTA_PACKED_CREDENTIALS."
+        fi
+
+}
+
+do_image_ostreepush[depends] += "fiotools-native:do_populate_sysroot ca-certificates-native:do_populate_sysroot"
+IMAGE_CMD_ostreepush_lmp () {
+	if [ -n "${USE_FIOTOOLS}" ]; then
+		bbwarn "Pushing repo by using fiopush"
```

Comment:
Better to remove as this will generate a build warning, which is not really what is going on (useful for debugging, but we can safely remove).

You can change to bbnote if needed, but need a run to see if that shows up at the build log.

Reply:
I tried `bbnote` and didn't see anything in the logs so decided to put `bbwarn` as these log messages are useful, we can see which type of `push` we use.

Reply:
Only problem is that we're trying to push the build to stop producing warnings and flag when a warning happens (we even produce a special file with all warnings, such as https://ci.foundries.io/projects/msul-dev01/lmp/builds/793/raspberrypi4-64/artifacts/other/bitbake_warning.log), and with this change every build will generate such warning (ones using fiotools).

Maybe just drop this line as we now got it to work and also tested the output?

Reply:
Yep, I think it makes sense to drop it now and just rely on the value `USE_FIOTOOLS` (which is printed in the logs) to see which of the tools was used by a specific CI run.

Reply:
@ricardosalveti I removed `bbwarn`.